src/pages/builder/index.tsx

```diff
@@ -142,31 +139,34 @@ export default function BuilderPage() {
                   onChange={(value) => handleSelectChange(value, 'workExperience')}
                   options={selectOptions}
                 />
-                <ProFormList
-                  name="users"
-                  creatorButtonProps={{
-                    position: 'bottom',
-                    creatorButtonText: 'Add new work experience'
-                  }}
-                  initialValue={[{ employer, position, dateTime, remark }]}
-                >
-                  <ProFormGroup key="group">
-                    <ProFormText name="employer" label="Your last employer" width="md" placeholder="EPAM" />
-                    <ProFormText
-                      name="position"
-                      label="Position on the job"
-                      width="md"
-                      placeholder="Senior Software Engineer"
-                    />
-                    <ProFormDateRangePicker name="dateTime" label="When did you work there" />
-                    <ProFormTextArea
-                      name="remark"
-                      label="Key points about this job"
-                      width="lg"
-                      placeholder="In short phrases tell us about what you did there"
-                    />
-                  </ProFormGroup>
-                </ProFormList>
+                {(cvDataFromRedux?.workExperience || []).map(({ companyName, position, dateTime, description }) => (
```

Comment:
Instead of mapping the initialValues with this component (which is not correct way of using this - you would have an 'Add' button for every work item), you could send the array to the initialValues